documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+
+The Waardepapieren solution should adhere to EN 301 549 / WCAG 2.0 though this is largely dependent on the integration of the solution in the user interfaces at the side of the issuer and not the solution itself. The project does involve the implementation of a reference validator app  / wallet however which must adhere to EN 301 549 / WCAG 2.0.
+
+As said, the solution also aims at a W3C verifiable credential compliant solution. This compliancy is deferred to the ongoing development of Discipl however. The solution adheres to the main mandatory concepts of this and related standards however.
+
+## Organisational layer
+
+The following stakeholders are identified:
+
+- issuer : automated (abundance) service available at location of issuer and/or through its website
+- clerk : employee that helps a client at location of issuer, services, identifies and authorizes the client manually
+- authorization service : automated service that identifies and authorizes client when client self services him/herself and accesses issuer through website
+- client : person in need of verifiable information required by a validator. It also becomes a holder of the resulting signed information.
+- validator : actor that needs to receive / verify information from client
+- administrator : actor that controls and monitors the automated issuer service
+
+An example of the essential conversation between real people as actors that would take place
+in the use cases the waardepapieren solution facilitates is given below:
+
+![](images/vcseq0.png)
+
+In essence this is the very proces that also takes place when retrieving a proof through the waardepapieren solution as can be seen below in the other verifiable claim sequence diagrams.
+
+![](images/vcseq.png)
+
+## Information layer
+
+All information is temporarily held in verifiable claim channels on the so called discipl ephemeral platform (http://github.com/discipl/discipl-core-ephemeral) in memory. This platform will enforce access rights denoted by the channel owner in specific allow claims within the channel, so by default all channels are only accessible to the channel owner holding a corresponding private key created and temporarily held by the channel owner.Of course the party hosting the ephemeral server and it's administrator needs to be trusted also, but as this is the issuer as source of the information it is by definition a party already trusted with this information by the client.
+
+The following channels will be created using this solution holding the following information (verifiable claims) with what kind of accessibility retention:
+
+1) public service channel of issuer, destroyed at service shutdown
+  - that a service with some id is attending to needs for what : (ssid, attendTo, what)
```

Comment:
This kind of sentence is not really friendly to the reader. Maybe we want to instead _italicize_ the parameters mentioned in the text? Possibly in addition to listing them at the end. (Goes for the rest of this section as well)

Reply:
I changed the list into a table which already makes it more readable. italicised the vc triplets